be/CMakeLists.txt

```diff
@@ -329,6 +329,14 @@ set_target_properties(opentelemetry_trace PROPERTIES IMPORTED_LOCATION ${THIRDPA
 add_library(opentelemetry_resources STATIC IMPORTED GLOBAL)
 set_target_properties(opentelemetry_resources PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib64/libopentelemetry_resources.a)
 
+add_library(avro STATIC IMPORTED GLOBAL)
+set_target_properties(avro PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib64/libavro.a)
```

Comment:
Add a related issued to correspond with the roadmap(https://github.com/StarRocks/starrocks/issues/16445)
![image](https://user-images.githubusercontent.com/44767902/212744283-93782034-3170-45d8-806a-67fe8b65defa.png)